client/src/pages/ProfileSetting.js

```diff
@@ -0,0 +1,108 @@
+import React from "react";
+import Onboarding from "../components/Onboarding";
+import Grid from '@material-ui/core/Grid';
+import Typography from '@material-ui/core/Typography'; 
+import TextField from '@material-ui/core/TextField';
+import InputAdornment from '@material-ui/core/InputAdornment';
+import DropdownSelect from '../components/DropdownSelect';
+
+const moment = require('moment-timezone');
```

Comment:
Because you're workin on the front-end with React, your ES6+ syntax will be automatically transpiled to work with older browsers, and because you're using the newer `import` statements above (to import React components), it would be better that you do the same importing `moment`.